### DISCUSSION TITLE
如何开启 音色 tab，我目前没有这个tab

### DISCUSSION THREAD

**Discussion by ByeLe** [lenML/Speech-AI-Forge#172] [问答 | Q&A]
官方截图：
![image](https://github.com/user-attachments/assets/590f7ae4-0888-44e6-8021-22e01ec93242)
我的截图：
![image](https://github.com/user-attachments/assets/e124f82f-dbb2-4817-b3e4-e99a96a112a0)

**Comment by zhzLuke96** [comment]
![image](https://github.com/user-attachments/assets/81a49611-732e-41b3-9f79-a0a585dee5e2)

最新的代码里，音色builder在这个位置

**Comment by zhzLuke96** [comment]
看你的截图版本是 0.5.5.1 应该还没有最新的这些功能，请先拉取最新的代码再试

**Reply by ByeLe** [reply]
有最新版本的整合包吗，跪求

**Reply by zhzLuke96** [reply]
> 有最新版本的整合包吗，跪求

不清楚你用的什么整合包，整合包应该是带有 git 的，所以你直接在项目目录下执行 `git pull` 就能更新到最新的代码，然后更新依赖环境应该就能用最新的代码

**Reply by ByeLe** [reply]
用的是windows版本的，我试试。
